Examples/FriendlyFlix/firebase.json

```diff
@@ -0,0 +1,31 @@
+{
+  "emulators": {
+    "auth": {
+      "port": 9099
+    },
+    "hosting": {
```

Comment:
Do you need hosting emulators for this project? 

Same question for auth?

Reply:
I think most of the hosting config and perhaps the ui: stuff needs to be removed. These aren't data connect related.

Reply:
Removed the hosting emulator, but I think we need auth.